src/Symfony/Component/Console/Terminal.php

```diff
@@ -17,6 +17,41 @@ class Terminal
     private static ?int $height = null;
     private static ?bool $stty = null;
 
+    /*
+     * About Ansi color types: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors
+     * For more information about true color support with terminals https://github.com/termstandard/colors/
+     */
+    public static function getTermColorSupport(): string
+    {
+        // Try with $COLORTERM first
+        if (\is_string($colorterm = getenv('COLORTERM'))) {
+            $colorterm = strtolower($colorterm);
+
+            if (str_contains($colorterm, 'truecolor')) {
+                return 'Ansi24';
+            }
+
+            if (str_contains($colorterm, '256color')) {
+                return 'Ansi8';
```

Comment:
This might be a good use case for an enum instead of returning magic strings (especially given that this is a public API, so we might have code using it outside Symfony)

Reply:
Totally agree. I was thinking about that. But Ennum support was really new in Simfony code base, and the contributing guide doesn't say anything about the conventions (I don't know in which namespace path put this one).
I will do it. And add comment for each enum case.

Reply:
I added the new enum, but I'm unsure about the path.
But more importantly, I have also moved a part of the logic to enum methods.